docs/docs/migration/migration-from-v7-to-v8.md

```diff
@@ -126,8 +128,55 @@ The class-validator peer dependency has been bumped to v0.14.0:
 npx @comet/upgrade v8/update-class-validator.ts
 ```
 
+#### Remove passport
```

Comment:
```suggestion
### Remove passport
```

This does more than upgrading peer dependencies section, so it should be a separate section.